src/UITests/README.md

```diff
@@ -0,0 +1,30 @@
+## UI Tests
```

Comment:
I think we should add info about how to run the test (e.g. running winappdriver)

Reply:
Info added